mirascope/core/base/_call_factory.py

```diff
@@ -318,6 +319,6 @@ def base_call(
             json_mode=json_mode,
             client=client,
             call_params=call_params,
-        )  # type: ignore
+        )  # pyright: ignore [reportReturnType, reportCallIssue]
```

Comment:
OK, I will think it

Reply:
Here, too, the types of other functions are now valid and the decision is made by Generics. It is difficult to fix because the response varies from provider to provider.